app/models.py

```diff
@@ -550,3 +557,21 @@ def bulk_lookup():
                 db.session.commit()
             except IntegrityError:
                 db.session.rollback()
+
+class Leaders(db.Model):
```

Comment:
...more...actually `deploy()` does not run `db.create_all()`, so you're right - views can just be generated with alembic in `upgrade()`.